### ISSUE TITLE
RFE: display UTC timestamps

### ISSUE THREAD

**Issue by GNiklasch** [Gravity-Spy/gravityspytools#21]
A suggestion for a possible enhancement of the Similarity Search and Display Your Collection With Metadata Info tools:

Both tools obviously have access to the (GPS) timestamps of the events underlying the Gravity Spy subjects. Would it be feasible to convert these timestamps to UTC and display them in an additional column of the table?

This would eliminate the need for clumsy workarounds involving transient ad-hoc collections with borrowed subjects to persuade the timeline histogram to reveal enough detail; and it would help with studying diurnal / weekly / monthly patterns, correlations with Livingston train schedules, correlations with activities recorded in the aLogs, etc. The capability to sort the table on the timestamps (by clicking on the column header) would come for free, as in the existing columns, when they're formatted reasonably (ISO8601 style like in the histogram labels: YYYY-MM-DD hh:mm:ss).

Cheers, Gerhard (ZngabitanT)

**Comment by UnbiasedBrigade** [comment]
I would agree here: the ability to easily access timestamps on subjects would be very useful for data collection about possible glitch classes, especially in finding possible triggers with time correlations, either to eachother or to events an the aLogs, as Zngabitan stated. This seems like it would be a very useful addition to the GS Tools.

Best wishes, UnbiasedBrigade
